lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1150,7 +1156,7 @@ public function get_item_schema() {
 		$base = $this->get_post_type_base( $this->post_type );
 		$schema = array(
 			'$schema'    => 'http://json-schema.org/draft-04/schema#',
-			'title'      => $base,
+			'title'      => $this->post_type,
```

Comment:
Should be fine for now. We'll eventually need to revisit these schemas to dot our i's and cross our t's


> On May 4, 2015, at 16:53, Joe Hoyle <notifications@github.com> wrote:
> 
> In lib/endpoints/class-wp-rest-posts-controller.php:
> 
> > @@ -1150,7 +1156,7 @@ public function get_item_schema() {
> >  		$base = $this->get_post_type_base( $this->post_type );
> >  		$schema = array(
> >  			'$schema'    => 'http://json-schema.org/draft-04/schema#',
> > -			'title'      => $base,
> > +			'title'      => $this->post_type,
> @danielbachhuber this was using posts before for the "title", however all our other endpoints use singular names. I presumed this title refers to the object name, not plural form, so i changed this to just the post type. Do you see issues with that?
> 
> —
> Reply to this email directly or view it on GitHub.
>